src/h.js

```diff
@@ -1,6 +1,8 @@
 import { VNode } from './vnode';
 import options from './options';
+import { iterableToArray, isFunction } from './util';
 
+const hasSymbolSupport = isFunction(Symbol) && Symbol.iterator;
```

Comment:
uh huh, I dont know how I have missed this comment. Its now failing in IE, lets fix it for next release.